src/phoenix/server/api/routers/v1/spans.py

```diff
@@ -71,6 +71,7 @@ async def query_spans_handler(request: Request) -> Response:
         or request.headers.get("project-name")
         or DEFAULT_PROJECT_NAME
     )
+    end_time = payload.get("end_time") or payload.get("stop_time")
```

Comment:
is this behaving the way we expect? I wonder if this line is the reason why tests are failing in CI

Reply:
Can we just maintain support for `stop_time` on the GET route only rather than adding support for it on the POST route? I don't think we actually need backward compatibility on the POST route since only up-to-date clients will use it.

Reply:
I think they share the same handler?

Reply:
I didn't migrate enough @anticorrelator